tracer/test/Datadog.Trace.Tests/Tagging/TagsListTests.cs

```diff
@@ -113,8 +116,11 @@ public void Serialization_ServiceEntrySpan()
         {
             var tracer = new Mock<IDatadogTracer>();
             var traceContext = new TraceContext(tracer.Object);
+
             var rootSpan = new Span(new SpanContext(SpanContext.None, traceContext, "service1"), start: null);
+            traceContext.AddSpan(rootSpan);
```

Comment:
NOTE: `Span.IsRootSpan` won't be `true` unless the span is added to the `TraceContext`.
We should look into improving this check (has parent or parentId = 0? parent context has spanId = 0?)

Reply:
That seems like a readonly property we can set at the creation of a span, because we should know the parent/child relationship at that point

Reply:
Good idea. I'll try and tackle this in a separate PR.